src/style.css

```
@@ -179,25 +178,14 @@ SEZIONE BREADCRUMB E FOOTER
 
 #breadcrumb, footer {
     background-color: var(--breadfooterbgcolor);
-    color:var(--breadfootercolor);
-}
-
-#breadcrumb a:link, footer a:link{
-    color: var(--linkbreadfooter);
-}
-
-#breadcrumb a:visited, footer a:visited{
-    color: var(--visitedbreadfooter);
+    color:var(--txtbreadfooter);
+    border-top: 2px solid var(--visitedcolor); 
+    border-bottom: 2px solid var(--visitedcolor); 
 }
 
 #breadcrumb {
     padding: 0.5em;
-    align-items: center;
-    text-align: center;
-}
-
-#breadcrumb p{
-    margin: 0;
+    margin-top: 0.3em;
 }
```

```suggestion
}

#breadcrumb p{
    margin: 0;
}
```
Rimettere come prima il "breadcrumb p"